js/ui/list/ui.list.base.js

```diff
@@ -246,6 +246,14 @@ export const ListBase = CollectionWidget.inherit({
                     pageLoadingText: '',
                     useInkRipple: true
                 }
+            },
+            {
+                device: function() {
+                    return devices.real().mac;
+                },
+                options: {
+                    pageLoadMode: 'scrollBottom'
```

Comment:
This is the default value for the mac as it is.
We need to specify the [default value for the documentation](https://github.com/DevExpress/DevExtreme/blob/21_1/js/ui/list.d.ts#L327-L328) (macbook is a desktop, but primarily a mac)

Reply:
oh thx